src/main/java/org/isf/accounting/gui/BillDataLoader.java

```diff
@@ -81,17 +84,26 @@ private List<Bill> mergeBillsFromPeriodAndFromPayments() {
 		return billAll;
 	}
 
-	private List<Bill> getClosedBills(String status) {
-		return billPeriod.stream()
+	private List<Bill> getClosedBills(String status, String username) {
+		List<Bill> 	list = billPeriod.stream()
 				.filter(bill -> bill.getStatus().equals(status))
 				.collect(Collectors.toList());
+		if (username != null) {
+			list = list.stream().filter(bill-> bill.getUser().equals(username)).collect(Collectors.toList());
+		}
+		return list;
 	}
 
-	private List<Bill> getPendingBills(String status) throws OHServiceException {
-		return patientParent != null ? billManager.getPendingBillsAffiliate(patientParent.getCode()) :
-				billPeriod.stream()
+	private List<Bill> getPendingBills(String status, String username) throws OHServiceException {
+		if (patientParent != null) return  billManager.getPendingBillsAffiliate(patientParent.getCode()) ; 
+			List<Bill> list = billPeriod.stream()
```

Comment:
Line 98 should really be
<pre>
if (patientParent != null) {
   return  billManager.getPendingBillsAffiliate(patientParent.getCode());
}
</pre>
 
And then the formatting starting with line 99 looks indented too much.

Reply:
Sorry, merged to fast, changes applied on develop directly. Thanks!